components/page.jsx

```diff
@@ -12,6 +12,20 @@ var DevRibbon = (process.env.NODE_ENV === 'production' &&
                 : require('./dev-ribbon.jsx');
 
 var Page = React.createClass({
+  statics: {
+    handlerForPage: function(router, url) {
+      return router.match(url).routes[1].handler;
+    },
+    titleForHandler: function(handler) {
+      var title = 'Mozilla Learning';
+
+      if (handler.pageTitle) {
+        title = handler.pageTitle + ' - ' + title;
```

Comment:
> @toolness I don't understand how `handler.pageTitle` is hooked up with w/e `pageTitle` is set on the actul page, e.g., https://github.com/mozilla/teach.webmaker.org/pull/779/files#diff-f61a39ec11b5b6b336c2b6fdc3a59c4bR76.  Guidance?  :cold_sweat: :wink: 

nvm I think I got it!  it's via `<Route name="about" path="/about/"
     handler={require('../pages/about.jsx')}/>` in `routes.jsx` :bowtie: